src/seedu/addressbook/Main.java

```diff
@@ -85,8 +87,11 @@ private void runCommandLoopUntilExitCommand() {
             CommandResult result = executeCommand(command);
             recordResult(result);
             ui.showResultToUser(result);
-
-        } while (!ExitCommand.isExit(command));
+            try {
+            	isFileExists();
```

Comment:
The abstraction level of this method seems to be too low for this location. If another method does things that would ultimately require interaction with the file, would this exception be caught? It also goes against the single responsibility principle. Consider whether this method, whose purpose to is to run commands should be caring about whether a file exists. Try finding a suitable location in storage file.